.github/workflows/ci.yml

```
@@ -16,21 +16,28 @@ jobs:
           - darwin
           - linux
           - win32
+        arch:
+          - x64
+          - arm64
         include:
           - name: darwin
-            os: macos-14
+            arch: x64
+            os: macos-13
+          - name: darwin
             arch: arm64
-            arch2: x64
+            os: macos-14
           - name: linux
             os: ubuntu-latest
-            arch: x64
-            # node-gyp --arch=arm64 still produced x64 binaries.
-            # Maybe use https://buildjet.com/for-github-actions/docs/about/pricing#arm?
-            arch2: none
           - name: win32
             os: windows-latest
-            arch: x64
-            arch2: arm64
+        exclude:
+          # node-gyp --arch=arm64 still produced x64 binaries.
+          # Maybe use https://buildjet.com/for-github-actions/docs/about/pricing#arm?
```

While at it, we could change this to:
```suggestion
          # See https://github.com/lydell/node-pty/issues/3
```